requirements_conda.yml

```diff
@@ -0,0 +1,17 @@
+name: desc-env
```

Comment:
Same as earlier comment: do new requirements have to be added to both `requirements.txt` and this file?

Reply:
[Yea](https://github.com/PlasmaControl/DESC/pull/261/files#r978885865)